x-pack/legacy/plugins/task_manager/task_runner.test.ts

```diff
@@ -202,6 +219,166 @@ describe('TaskManagerRunner', () => {
     sinon.assert.calledWithMatch(logger.warning, /not cancellable/);
   });
 
+  test('sets startedAt, status, attempts and retryAt when claiming a task', async () => {
+    const timeoutMinutes = 1;
+    const id = _.random(1, 20).toString();
+    const initialAttempts = _.random(0, 2);
+    const { runner, store } = testOpts({
+      instance: {
+        id,
+        attempts: initialAttempts,
+        interval: undefined,
+      },
+      definitions: {
+        bar: {
+          timeout: `${timeoutMinutes}m`,
+          createTaskRunner: () => ({
+            run: async () => undefined,
+          }),
+        },
+      },
+    });
+
+    await runner.claimOwnership();
+
+    sinon.assert.calledOnce(store.update);
+    const instance = store.update.args[0][0];
+
+    expect(instance.attempts).toEqual(initialAttempts + 1);
+    expect(instance.status).toBe('running');
+    expect(instance.startedAt.getTime()).toEqual(Date.now());
```

Comment:
Since we're mocking `Date.now()`, it will always return the mocked date. The task runner uses `Date.now()` at this line https://github.com/elastic/kibana/pull/39829/files#diff-77f0722c8f1b773af3bea53797d6f346R170.

Reply:
Update: Now that we're using fake timers, both dates are guaranteed to be the same.